Dockerfile

```diff
@@ -4,15 +4,17 @@ FROM python:3.7.11
 RUN apt update && apt -y install curl
 
 # Installing Node
-RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - 
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
 RUN apt-get install -y nodejs
 
+RUN npm i -g npm
```

Comment:
Upgrades npm as we were having lots of warnings on using an old, incompatible, version with the package-lock.json lockfile.